deployment/components/common/worker-mounts.yaml

```
@@ -4,6 +4,9 @@
   - name: host-boot
     hostPath:
       path: "/boot"
+  - name: host-swap
```

This is different name than what is used in the volumeMounts below. I also suggest to use the same naming than in the Helm chart 

```suggestion
  - name: host-proc-swaps
```